next.config.js

```diff
@@ -1,3 +1,9 @@
+const baseUrl = process.env.VERCEL_ENV
+  ? `https://${process.env.VERCEL_URL}`
+  : 'http://localhost:3000'
```

Comment:
Should we have this as its own env variable incase someone hosts it at a different port?